lib/protobuf/active_record/transformer.rb

```diff
@@ -15,6 +15,7 @@ def call(proto)
       def nullify?(proto)
         return false unless options[:nullify_on]
         return false unless proto.field?(:nullify) && proto.nullify.is_a?(Array)
+        return false if proto.nullify.empty?
```

Comment:
Looks like `proto.nullify` isn't guaranteed. Let's add a guard in for `proto.field?(:nullify)`.